### PR TITLE
405 network filter moved to mapControls

### PR DESCRIPTION
**Pull request by ahjyrkia**
closes https://github.com/HSLdevcom/jore-map-ui/issues/405

### CODE REVIEW

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 1]
```diff
@@ -33,28 +36,78 @@ export default class MapLayersControl extends React.Component
         });
     }
 
+    public toggleTransitType = (type: TransitType) => {
+        NetworkStore.toggleTransitType(type);
+    }
+
+    public toggleIsLinksVisible = () => {
```
Comment (neodyymi) [inline src/components/map/mapControls/MapLayersControl.tsx]: These names feel like bad english 😅 
Could they just be `toggleLinksVisibility` etc

Comment (neodyymi) [reply src/components/map/mapControls/MapLayersControl.tsx]: I'm also not sure why these are wrapped in local functions when the ones from networkstore are selfexplanatory already..

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 2]
```diff
@@ -33,28 +36,78 @@ export default class MapLayersControl extends React.Component
...
-                        onClick={this.toggleRadioButton(option.SATELLITE)}
-                        checked={this.state.selectedOption === option.SATELLITE}
-                        text={option.SATELLITE}
-                    />
-                    <RadioButton
-                        onClick={this.toggleRadioButton(option.TERRAIN)}
-                        checked={this.state.selectedOption === option.TERRAIN}
-                        text={option.TERRAIN}
-                    />
+                    <div className={s.flexColumn}>
```
Comment (neodyymi) [inline src/components/map/mapControls/MapLayersControl.tsx]: This div seems to be redundant.

File: src/components/map/mapControls/mapLayersControl.scss [hunk 3]
```diff
@@ -24,6 +25,31 @@
         border: 1px solid $darkGrey;
         z-index: 1;
         padding: 10px;
+
```
Comment (neodyymi) [inline src/components/map/mapControls/mapLayersControl.scss]: There's a non-breaking space after `.mapLayersContainer`. Seems like it's been there before, but I think it should be removed asap.

Comment (ahjyrkia) [reply src/components/map/mapControls/mapLayersControl.scss]: Tried removing it. Didn't work. Added it back.

**Comment by neodyymi** [comment]
Add the weird behaviour of the hover on some browsers as an issue and this is fine

**Pull request merged**